### DISCUSSION TITLE
How to make the sidebar in a dashboard collapsed by default?

### DISCUSSION THREAD

**Discussion by pensivedog** [quarto-dev/quarto-cli#11144] [Q&A] [answered]
### Description

I have a quarto website that contains several dashboards, on some of which I'd like to have sidebars. I've searched and can't find a working way to have the sidebars collapsed by default. I've tried the following, which does not work:

```
format:
  dashboard:
    sidebar:
      collapse: true
```

Is there any way to do something like the above or, even better, something like the below?

```
# {.sidebar data-collapsed="true" data-open="collapsed"}
```

**Comment by cderv** [comment, marked as answer]
We currently don't allow to customize the default state. It will be expanded by default 
https://github.com/quarto-dev/quarto-cli/blob/a2a22e5cd7d4ea5300b8f9ca1d094d998f2885f8/src/format/dashboard/format-dashboard-sidebar.ts#L129-L143

This means this is a feature request idea. 

In the meantime you may be able to use JS on document loaded to change the default state of your sidebar I guess. But I don't know specifically how... 🤷‍♂️ Sorry.

**Reply by pensivedog** [reply]
Thank you for the reply. I tried to do this with JS but could only make it work by toggling it to closed on page load, which causes a rather unpleasant FOUC. I'll add it as a feature request.

**Reply by pensivedog** [reply]
Feature request #11165 created.
